immersion/kubeflow_pipelines/cicd/solutions/trainer_image_vertex/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/deeplearning-platform-release/base-cpu
+RUN pip install -U fire cloudml-hypertune scikit-learn==0.20.4 pandas==0.24.2
+WORKDIR /app
+COPY train.py .
+
+ENTRYPOINT ["python", "train.py"]
```

Comment:
My build fails here with the below error, I guess when trying to import `sklearn.pipeline`. Could it be an `sklearn` version issue? I do see `sklearn` downgrading to the requested `0.20.4`

```
ModuleNotFoundError: No module named 'pipeline'
```